lib/src/ffmpeg/ffmpeg_command.dart

```diff
@@ -7,9 +7,12 @@ import 'package:ffmpeg_cli/src/ffmpeg/log_level.dart';
 /// Executes FFMPEG commands from Dart.
 class Ffmpeg {
   /// Executes the given [command].
-  Future<Process> run(FfmpegCommand command) {
+  ///
+  /// Provide a [ffmpegPath] to customize the path of the ffmpeg cli.
```

Comment:
Please include a couple of examples so that the use is crystal clear. For example, please include what such a path might look like on a Mac, and then what a path might look like on Windows. I would imagine that at a minimum the file path separators would be reversed, and it's nice to make that really clear to users.

Reply:
Updated.